src/_includes/layout.njk

```diff
@@ -147,22 +147,22 @@
           <div class="col-md-6">
             <ul class="list-inline social-buttons">
               <li class="list-inline-item">
-                <a href="https://www.facebook.com/codeconnector" target="blank">
+                <a href="https://www.facebook.com/codeconnector" target="blank" aria-label="Code Connector's Facebook navigation link">
                   <i class="fab fa-facebook-f"></i>
                 </a>
-              </li>
+              </li> 
               <li class="list-inline-item">
-                <a href="https://twitter.com/codeconnector_" target="blank">
-                  <i class="fab fa-twitter"></i>
+                <a href="https://twitter.com/codeconnector_" target="blank" aria-label="Code Connector's Twitter navigation link">
+                  <i class="fab fa-twitter" title="Navigate to Twitter"></i>
                 </a>
               </li>
               <li class="list-inline-item">
-                <a href="https://www.linkedin.com/company/code-connective/" target="blank">
+                <a href="https://www.linkedin.com/company/code-connective/" target="blank" aria-label="Code Connector's LinkedIn navigation link">
                   <i class="fab fa-linkedin-in"></i>
                 </a>
               </li>
               <li class="list-inline-item">
-                <a href="https://www.eventbrite.com/o/code-connector-15650145492" target="blank">
+                <a href="https://www.eventbrite.com/o/code-connector-15650145492" target="blank" aria-label="Code Connector's Eventbrite navigation link">
```

Comment:
Screen readers will announce that this is a link, so a better label would just be "Code Connector's Eventbrite" or "Eventbrite"

Reply:
Fixed